Sources/PubNub/Subscription/SubscriptionSession.swift

```diff
@@ -33,7 +33,7 @@ public class SubscriptionSession {
 
   public let uuid = UUID()
   let longPollingSession: SessionReplaceable
-  internal(set) var configuration: SubscriptionConfiguration
+  var configuration: SubscriptionConfiguration
```

Comment:
Does it have to change?

Reply:
@kleewho this one changed in master already.